app/controllers/signup.js

```diff
@@ -65,110 +69,110 @@ export default class SignupController extends Controller {
       errorMessage: 'Valid Email is required',
       required: true,
       showError: false,
-      validator: this.emailValidator
+      validator: this.emailValidator,
     },
     {
-      id: 'phoneNumber',
+      id: 'phone_number',
```

Comment:
The `id` should be just `phone` now, right?

Reply:
yeah, changed!